lib/gtfs/direction.ex

```diff
@@ -1,7 +1,50 @@
 defmodule Gtfs.Direction do
+  alias Gtfs.{Csv, Route}
+
   @type id :: 0 | 1
 
-  @spec from_string(String.t()) :: id()
-  def from_string("0"), do: 0
-  def from_string("1"), do: 1
+  @type t :: %__MODULE__{
```

Comment:
In GTFS, `direction.txt` is separate from `routes.txt` because all the data has to fit into tables. Since we can do arbitrarily shaped data, it might be simpler just to have the `direction_name` field directly in the `route` struct, without any kind of `%Direction{}` struct, like how it's stored in the typescript route type.

```
%Route{
  id: "57",
  direction_names: %{
    0 => "Outbound",
    1 => "Inbound",
  }
}
```

Reply:
Well we mostly have that, we just retain all the direction information in Route. We could simplify, but that would make it much more annoying if we ever did end up wanting to use `direction_destination` in the future.

Reply:
The `route_id` and `direction_id` fields are redundant with the map keys, so we wouldn't need to store them (similar to how we don't store trip_id in the stop_time struct). If we remove them, then `direction_name` is the only field left in the struct.

We've never talked about `direction_destination` as even a potential feature. Trip headsigns serve that same purpose. So I'd lean towards simplicity now, even if it makes adding `direction_destination` later harder.

Reply:
I don't think you can pair it down that much. You need to maintain route ID in order to assign it to the correct route.

Reply:
Oh I'm sorry, you are suggesting that in the route. That's fine, that can work, but can we merge this and do that refactoring later so this isn't held up for it?

Reply:
Added a ticket for the refactor: https://app.asana.com/0/1112935048846093/1125751680948291